### ISSUE TITLE
Deprecation warnings for faker.random.number

### ISSUE THREAD

**Issue by ryanwilsonperkin** [Shopify/quilt#2153]
## Overview

The use of `faker.random.number` in `graphql-fixtures` is resulting in deprecation warnings in the console, it should be updated to use `faker.datatype.number`

```
Deprecation Warning: faker.random.number is now located in faker.datatype.number
    at Random.number (node_modules/@faker-js/faker/lib/random.js:37:13)
```

## Checklist

- [x] Please delete the labels section before submitting your issue
- [x] I have described this issue in a way that is actionable (if possible)

**Comment by BPScott** [comment]
This is turning into a stupid game of whack-a-mole. We've already had 2 PRs for deprecated faker apis - https://github.com/Shopify/quilt/pull/2140 and https://github.com/Shopify/quilt/pull/2141. Can we make sure we get all of them this time?

**Comment by ryanwilsonperkin** [comment]
Agreed, we can raise on deprecation warnings in our test suite for this package which should allow us to catch any of these that are slipping through. I'm going to tackle this momentarily

**Comment by BPScott** [comment]
I've searched through the `@faker-js/faker` package for console.log statements.

 https://unpkg.com/browse/@faker-js/faker@5.5.3/dist/faker.js suggests that `faker.random.{number,float,uuid,boolean, hexaDecimal}` are all deprecated, and should be accessed as `faker.datatype.BLAH` instead. There's no other relevant files that contain deprecations (faker is distributed in a few different ways, so these console.logs are repeated in a few files).
